wordpress-mobile/WordPress-Android
# Stats revamp v2 Follower Totals card

Fixes #16240

This PR adds new Follower Total Cards to the stats screen. This new card will replace the old follower total cards if the feature flag is enabled.

A new stats URL, "summary" is added on FluxC.

TODO:
- [x] [Corresponding FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2375) needs to be merged before merging this
- [x] [Corresponding FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2377) needs to be merged before merging this

Known issues:
- Vertical padding of the card. It'll be fixed in another PR for all new cards.
- Style of the title. It'll be fixed in another PR.

|before|after|
|-|-|
|<img width="320" alt="Screen Shot 2022-05-01 at 15 37 07" src="https://user-images.githubusercontent.com/2471769/166146253-9870980d-6e5f-42bd-99bc-8b987923a6a0.png">|<img width="320" alt="Screen Shot 2022-05-01 at 15 36 03" src="https://user-images.githubusercontent.com/2471769/166146256-061c867a-7b9e-4c0c-9ed2-8b1702c0f492.png">|

To test:
1. Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings).
2. Select Debug Settings.
3. Find StatsRevampV2FeatureConfig under Features in development enable it and restart app.
4. Go to stats either using quick links or menu.
5. Ensure that it opens in Insights tab otherwise switch to Insights tab.
6. If the Follower Totals card is not present, go to the stats management (either through ⚙️ menu at the top right hand corner or Add new stats card at the bottom) and add the card and save, then return to Insights.
7. Verify that the new Follower totals card is visible and matches the new design.
8. Check your total followers count from web or old card and verify that the data in the new design is correct.

## Regression Notes
1. Potential unintended areas of impact
Current follower totals card

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested, it's not changed as it should be.

3. What automated tests I added (or what prevented me from doing so)
`FollowerTotalsUseCaseTest`

PR submission checklist:
